ee/tabby-webserver/src/schema/dao.rs

```diff
@@ -54,3 +57,13 @@ impl From<GithubOAuthCredentialDAO> for OAuthCredential {
         }
     }
 }
+
+impl From<EmailServiceCredentialDAO> for EmailServiceCredential {
+    fn from(value: EmailServiceCredentialDAO) -> Self {
```

Comment:
can we add a flag to email service credential dao for active flag? so we can disable email service as well

Reply:
Since it's stored as an Option, do we just want to use its presence or absence to indicate whether it's enabled? Or would we ever need to store the credentials while it's inactive?

Reply:
good point, I'm fine to add an delete interface to make it represents "inactive" (as we don't really need to store the credentials)

Once you done with this, could you also refactor https://github.com/TabbyML/tabby/blob/main/ee/tabby-db/src/github_oauth_credential.rs#L31 to making "delete" representing inactive?